app/services/profile_fields/add_base_fields.rb

```diff
@@ -0,0 +1,11 @@
+module ProfileFields
+  class AddBaseFields
+    include FieldDefinition
```

Comment:
For people who reviewed the draft PR previously: I switched this from inheritance to a mixin, as this isn't really "X is a Y" but "X behaves like a Y".